### PR TITLE
Update Swedish locale

### PR DESCRIPTION
**Pull request by alexandernanberg**
🇸🇪

### CODE REVIEW

**Comment by kossnocorp** [comment]
@alexandernanberg thank you 🙏 Please tell me when it's ready to be shipped and I'll release a new version with it.

**Comment by alexandernanberg** [comment]
@kossnocorp This should be good to go. I've double checked the translations with a copywriter so they should be accurate. There are some tests that are still skipped, but they were not really relevant to swedish (am/pm stuff)

**Comment by kossnocorp** [comment]
@alexandernanberg great!

**Comment by kossnocorp** [comment]
Thank you for your work, you did an amazing job! The PR was released as `v2.0.0-alpha.7`. See change log: https://gist.github.com/kossnocorp/a307a464760b405bb78ef5020a4ab136#v200-alpha7

File: src/locale/sv/test.js [hunk 1]
```diff
@@ -41,122 +41,124 @@ describe.skip('sv locale', function () {
...
-        var result = format(date, 'Mo [month]', {locale: locale})
-        assert(result === '4th month')
+        var result = format(date, 'Mo [månaden]', {locale: locale})
+        assert(result === '4:e månaden')
       })
 
       it('MMM', function () {
         var result = format(date, 'MMM', {locale: locale})
-        assert(result === 'Apr')
+        console.log(result)
```
Comment (artem-malko) [inline src/locale/sv/test.js]: Ooops

Comment (kossnocorp) [reply src/locale/sv/test.js]: 😱

Comment (alexandernanberg) [reply src/locale/sv/test.js]: Woops! Want me to fix it? 😅

Comment (kossnocorp) [reply src/locale/sv/test.js]: @alexandernanberg no worries, I'll do it

**Pull request merged**